### PR TITLE
feat: Create our own end point for video recording using BBB API

### PR DESCRIPTION
**Pull request by sachinchauhan2889**
This PR create Video-Recording Table and store data in it whenever get-recording end point is called.

### CODE REVIEW

File: app/api/video_recordings.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        channel = stream.channel
+        bbb = BigBlueButton(channel.api_url, channel.api_key)
+        result = bbb.request('getRecordings', params)
+
+        if result.data['response']['recordings']:
+            recordings = []
+            if type(result.data['response']['recordings']['recording']) is list:
+                recordings = result.data['response']['recordings']['recording']
+            else:
+                recordings.append(result.data['response']['recordings']['recording'])
```
Comment (MeghalBisht) [inline app/api/video_recordings.py:38]: @sachinchauhan2889 why are we checking for list here, I am not understanding this?

**Review by MeghalBisht — COMMENTED** [review]
down revision needs to be fixed, rest is working fine for me.

**Comment by sachinchauhan2889** [comment]
@MeghalBisht  down_revision has been updated. Now you can approve this.

**Review by MeghalBisht — DISMISSED** [review]
Works for me.

**Review by ranjitkshah — DISMISSED** [review]
works!

**Review by maze-runnar — APPROVED** [review]

**Review by MeghalBisht — APPROVED** [review]

**Pull request merged**